perf/lua/benchmark.lua

```diff
@@ -0,0 +1,63 @@
+local json = require('json')
+
+local M = {}
+
+local function format_report(bench)
+    local output_format = bench.output_format
+    local results = bench.results
+    local report = ''
+    if output_format == 'json' then
```

Comment:
This will not work with `output_format` equal to `JSON`. I would use `string.lower(output_format)` to avoid this. The same everywhere where you compare `output_format` with constants.

Reply:
Fixed at the moment of initialization, thanks!

Reply:
Thanks!